# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-shapes/shape-outside/values/shape-outside-polygon-004.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/css/css-shapes/shape-outside/values/shape-outside-polygon-004.html.ini
--- a/testing/web-platform/meta/css/css-shapes/shape-outside/values/shape-outside-polygon-004.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/css/css-shapes/shape-outside/values/shape-outside-polygon-004.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,18 +1,18 @@
 [shape-outside-polygon-004.html]
   [Three vertices - ex ex, ex ex, ex ex - computed]
     expected:
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
       if os == "linux": FAIL
-      if os == "mac" and not debug: ["FAIL", "PASS"]
 
   [Three vertices - em em, em em, ex ex - computed]
     expected:
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
       if os == "linux": FAIL
-      if os == "mac" and not debug: ["FAIL", "PASS"]
 
   [Three vertices - ex ex, em, em, em em - computed]
     expected:
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
       if os == "linux": FAIL
-      if os == "mac" and not debug: ["FAIL", "PASS"]
 
   [Two vertices - cm cm, cm cm - computed]
     expected: FAIL
@@ -71,28 +71,29 @@
 
   [One vertex - rem ch - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
 
   [Two vertices - rem rem, ch ch - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
 
   [Two vertices - ch ch, rem rem - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
 
   [Two vertices - rem ch, ch rem - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
 
   [Three vertices - ch ch, ch ch, ch ch - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
 
   [Three vertices - rem rem, ch ch, rem rem - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
 
   [Three vertices - rem rem, rem, rem, ch ch - computed]
     expected:
-      if os == "mac" and not debug: ["FAIL", "PASS"]
+      if (os == "mac") and not debug: ["PASS", "FAIL"]
+